suse_migration_services/migration_config.py

```diff
@@ -56,3 +58,24 @@ def get_migration_product(self):
             raise DistMigrationProductNotFoundException(message)
 
         return migration_product
+
+    def update_migration_config_file(self):
+        """
+        Update the default migration configuration with custom values
+        """
+        migration_custom_file = Defaults.get_system_migration_custom_config_file()
+        if os.path.exists(migration_custom_file):
+            with open(migration_custom_file, 'r') as custom_config:
+                new_config = yaml.safe_load(custom_config)
+                self.config_data.update(new_config)
+
+            default_migration_config_file = Defaults.get_migration_config_file()
+            with open(default_migration_config_file, 'w') as config:
+                yaml.dump(self.config_data, config, default_flow_style=False)
+                message = dedent('''
+                The migration file '{0}' has been updated with '{1}' info
+                ''')
+                log.info(message.format(
+                    default_migration_config_file,
+                    migration_custom_file
+                ))
```

Comment:
great now :+1: just nitpick on the formatting. Can we change this to

```python
message = dedent('''
    The migration file '{0}' has been updated with '{1}' info
''')
log.info(
    message.format(
        default_migration_config_file, migration_custom_file
    ).lstrip()
)
```

Reply:
I will leave it up to you since it's only a style issue. Usually one avoids multiple opening/closing brackets in one line though